tests/test_yaml2sbml.py

```diff
@@ -76,6 +76,26 @@ def test_yaml_import_observables(self):
 
         os.remove(sbml_test_dir)
 
+    def test_catch_invalid_sbml_identifier(self):
+        """
+        Test checks for invalid SBML identifiers.
+        """
+        yaml_dir = os.path.join(self.test_folder,
+                                'ode_input_invalid_SBML_identifier.yaml')
+        with self.assertRaises(RuntimeError):
+            _parse_yaml(yaml_dir,
+                        'Test_Model')
+
+    def test_catch_invalid_math(self):
+        """
+        Test checks for strings, that parseL3Formula can not parse.
```

Comment:
```suggestion
        Check for strings that parseL3Formula can not parse.
```
imperative / infinitive is usually preferred in docstrings